proto/frequenz/api/common/v1/microgrid/components/components.proto

```diff
@@ -141,8 +142,29 @@ message Component {
 
   // The operational lifetime of the component.
   frequenz.api.common.v1.microgrid.Lifetime operational_lifetime = 9;
+
+  // List of rated bounds present for the component identified by Metric.
+  repeated ComponentRatedBound rated_bounds = 10;
 }
 
+// Defines a components rated bound for a specific metric.
+message ComponentRatedBound {
+  // Metric type the rated bounds are for
+  frequenz.api.common.v1.metrics.Metric metric = 1;
+
+  // The maximum discharge and charge values for the device as
+  // rated by the manufacturer.
+  //
+  // Max discharge is contained in `lower`.
+  // Max charge is contained in `upper`.
+  //
+  // Max discharge (`lower`) is negative, as per passive sign convention.
```

Comment:
I'd suggest making the documentation more general, something along the lines of the following:

"The rated bounds of the specified metric. These indicate the lower and upper bounds of a metric, according to the component manufacturer's ratings."

Reply:
Updated